src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -197,6 +297,23 @@ def __enter__(self):
         self._inputs_queue = Queue()
         self._n_process = self._determine_worker_count()
 
+        if self._use_fork:
```

Comment:
It is tricky that `use_fork` and `not use_fork` are in different places, is it necessary?

Reply:
fixed